--- a/bin/generate_configuration
+++ b/bin/generate_configuration
@@ -16,7 +16,7 @@
 
 Log::Log4perl->easy_init($DEBUG);
 
-my $CONFIG_FILE = "$RealBin/../etc/agent_configuration.conf";
+my $CONFIG_FILE = "/etc/perfsonar/agentconfiguration.conf";
 my $ADDRESS;
 my $LOGGER_CONF;
 my $DEBUGFLAG;
--- a/bin/generate_gui_configuration
+++ b/bin/generate_gui_configuration
@@ -16,7 +16,7 @@
 
 Log::Log4perl->easy_init($DEBUG);
 
-my $CONFIG_FILE = "$RealBin/../etc/gui_agent_configuration.conf";
+my $CONFIG_FILE = "/etc/perfsonar/guiagentconfiguration.conf";
 my $LOGGER_CONF;
 my $DEBUGFLAG;
 my $HELP;
--- a/scripts/cron-mesh_config_agent
+++ b/scripts/cron-mesh_config_agent
@@ -1,2 +1,2 @@
 MAILTO=root
-0 2 * * * perfsonar /opt/perfsonar_ps/mesh_config/bin/generate_configuration &> /var/log/perfsonar/mesh_configuration_agent.log
+0 2 * * * perfsonar /usr/lib/perfsonar/bin/generate_configuration &> /var/log/perfsonar/mesh_configuration_agent.log
--- a/scripts/cron-mesh_config_gui_agent
+++ b/scripts/cron-mesh_config_gui_agent
@@ -1,2 +1,2 @@
 MAILTO=root
-0 2 * * * perfsonar /opt/perfsonar_ps/mesh_config/bin/generate_gui_configuration &> /var/log/perfsonar/mesh_configuration_gui_agent.log
+0 2 * * * perfsonar /usr/lib/perfsonar/bin/generate_gui_configuration &> /var/log/perfsonar/mesh_configuration_gui_agent.log
--- a/etc/gui_agent_configuration.conf
+++ b/etc/gui_agent_configuration.conf
@@ -75,14 +75,14 @@
 # the mesh, these will be used.
 <maddash_options>
     <perfsonarbuoy/owamp>
-        check_command            /opt/perfsonar_ps/nagios/bin/check_owdelay.pl
+        check_command            /usr/lib/perfsonar/bin/check_owdelay.pl
         check_interval           1800
         check_time_range         900
         acceptable_loss_rate     0
         critical_loss_rate       0.01
     </owamp>
     <perfsonarbuoy/bwctl>
-        check_command            /opt/perfsonar_ps/nagios/bin/check_throughput.pl
+        check_command            /usr/lib/perfsonar/bin/check_throughput.pl
         check_interval           28800
         check_time_range         86400
         acceptable_throughput    900
--- a/lib/perfSONAR_PS/MeshConfig/Agent.pm
+++ b/lib/perfSONAR_PS/MeshConfig/Agent.pm
@@ -37,7 +37,7 @@
 
 has 'meshes'                 => (is => 'rw', isa => 'ArrayRef[HashRef]', default => sub { [] });
 
-has 'regular_testing_conf'   => (is => 'rw', isa => 'Str', default => "/opt/perfsonar_ps/regular_testing/etc/regular_testing.conf");
+has 'regular_testing_conf'   => (is => 'rw', isa => 'Str', default => "/etc/perfsonar/regulartesting.conf");
 has 'force_bwctl_owamp'      => (is => 'rw', isa => 'Bool', default => 0);
 
 has 'addresses'              => (is => 'rw', isa => 'ArrayRef[Str]');
--- a/lib/perfSONAR_PS/MeshConfig/Generators/MaDDash.pm
+++ b/lib/perfSONAR_PS/MeshConfig/Generators/MaDDash.pm
@@ -525,7 +525,7 @@
 
 my %maddash_default_check_options = (
     "perfsonarbuoy/owamp" => {
-        check_command => "/opt/perfsonar_ps/nagios/bin/check_owdelay.pl",
+        check_command => "/usr/lib/perfsonar/bin/check_owdelay.pl",
         check_interval => 1800,
         check_time_range => 900,
         acceptable_loss_rate => 0,
@@ -534,7 +534,7 @@
         graph_url => '/serviceTest/graphWidget.cgi'
     },
     "perfsonarbuoy/bwctl" => {
-        check_command => "/opt/perfsonar_ps/nagios/bin/check_throughput.pl",
+        check_command => "/usr/lib/perfsonar/bin/check_throughput.pl",
         check_interval => 28800,
         check_time_range => 86400,
         acceptable_throughput => 900,
