--- a/etc/meshconfig-guiagent-logger.conf
+++ b/etc/meshconfig-guiagent-logger.conf
@@ -11,7 +11,7 @@
 log4perl.appender.A1=Log::Dispatch::FileRotate
 
   # alter location of the log file below
-log4perl.appender.A1.filename=/var/log/maddash/meshconfig-guiagent.log
+log4perl.appender.A1.filename=/var/log/perfsonar/meshconfig-guiagent.log
 log4perl.appender.A1.max=7
 log4perl.appender.A1.size=16777216
 log4perl.appender.A1.mode=append
